AnkiDroid/src/test/java/com/ichi2/anki/RobolectricTest.java

```diff
@@ -91,4 +95,23 @@ protected JSONObject getCurrentDatabaseModelCopy(String modelName) throws JSONEx
         Models collectionModels = getCol().getModels();
         return new JSONObject(collectionModels.byName(modelName).toString().trim());
     }
+
+    protected <T extends AnkiActivity> T startActivityNormallyOpenCollectionWithIntent(Class<T> clazz, Intent i) {
+        ActivityController multimediaController = Robolectric.buildActivity(clazz, i)
+                .create().start().resume().visible();
+        //noinspection unchecked
+        return (T) multimediaController.get();
+    }
```

Comment:
On first read I thought this was bound to the multimedia work but it looks like you've made a fully general purpose activity starter?

```suggestion
    protected <T extends AnkiActivity> T startActivityNormallyOpenCollectionWithIntent(Class<T> clazz, Intent i) {
        ActivityController controller = Robolectric.buildActivity(clazz, i)
                .create().start().resume().visible();
        //noinspection unchecked
        return (T) controller.get();
    }
```